### ISSUE TITLE
Last Vite update breaks the site

### ISSUE THREAD

**Issue by jan-dh** [nystudio107/craft-vite#25]
### Describe the bug
Web page now only shows
```
[
    'resources/js/head.js' => [
        'type' => 'file'
        'url' => 'head.efc7db5e.js'
        'options' => [
            'type' => 'module'
            'crossorigin' => true
            'onload' => 'e=new CustomEvent(\'vite-script-loaded\', {detail:{path: \'resources/js/head.js\'}});document.dispatchEvent(e);'
        ]
    ]
    'vendor.e2e1bd24.js' => [
        'crossorigin' => true
        'type' => 'import'
        'url' => 'vendor.e2e1bd24.js'
        'integrity' => ''
    ]
]
```

### To reproduce
Steps to reproduce the behaviour:
1. Update to last version
2. Refresh page
3. See error message 
```
[
    'resources/js/head.js' => [
        'type' => 'file'
        'url' => 'head.efc7db5e.js'
        'options' => [
            'type' => 'module'
            'crossorigin' => true
            'onload' => 'e=new CustomEvent(\'vite-script-loaded\', {detail:{path: \'resources/js/head.js\'}});document.dispatchEvent(e);'
        ]
    ]
    'vendor.e2e1bd24.js' => [
        'crossorigin' => true
        'type' => 'import'
        'url' => 'vendor.e2e1bd24.js'
        'integrity' => ''
    ]
]
```

### Expected behaviour
It doesn't break my website 😅

### Screenshots
![image](https://user-images.githubusercontent.com/5801781/150516247-ad3bacd7-fd82-4746-8699-619e2ac92189.png)

### Versions
- Plugin version: 1.0.22
- Craft version: 3.7.30

**Comment by frontedbe** [comment]
Can confirm. And not sure which versions of this plugin and/or Craft, vite.js to rollback to in order to get it working again..

**Comment by khalwat** [comment]
Fixed in 1.0.23 -> https://github.com/nystudio107/craft-vite/releases/tag/1.0.23
